### ISSUE TITLE
Can't compile on 4.26

### ISSUE THREAD

**Issue by JohnnyUrosevic** [ProjectBorealis/PBCharacterMovement#22]
I'll try to fix this linker error on my own, I'm just not sure what the issue is

This is the error I'm getting

```
ld.lld: error: undefined symbol: Z_Construct_UEnum_PhysicsCore_EPhysicalSurface()
>>> referenced by Module.PBCharacterMovement.gen.cpp
```

**Comment by mastercoms** [comment]
Seems like this enum was moved in 4.26, so you'll have to include a new public dependency module and possibly change include headers for it. I can't help you much further since we are not expecting to move to 4.26 for the time being.

**Comment by JohnnyUrosevic** [comment]
Do you know where it was moved? I can't seem to find it in the documentation

**Comment by mastercoms** [comment]
I do not, sorry.

**Comment by ZioYuri78** [comment]
The EPhysicalSurface enum is in the \UE_4.26\Engine\Source\Runtime\PhysicsCore\Public\Chaos\ChaosEngineInterface.h header but in theory you don't need to include the Chaos module for this.

Are you using the launcher binary or the github code of the engine?

**Comment by ZioYuri78** [comment]
> 
> 
> The EPhysicalSurface enum is in the \UE_4.26\Engine\Source\Runtime\PhysicsCore\Public\Chaos\ChaosEngineInterface.h header but in theory you don't need to include the Chaos module for this but i'm not 100% sure.
> 
> Are you using the launcher binary or the github code of the engine?

Ok, found the solution, you need to include the "PhysicsCore" module.
Just a heads up, when you have problems on compile to a new version of the engine go look in the forum, there is always a "C++ Transition Guide for x.xx" thread about what change in the code. 

https://forums.unrealengine.com/development-discussion/c-gameplay-programming/1838729-c-transition-guide-for-4-26

**Comment by JohnnyUrosevic** [comment]
The fix suggested works, are you guys interested in a pull request for this?

**Comment by mastercoms** [comment]
Sure!

**Comment by AndrewLish** [comment]
Hey, I am interested in this if you can.

**Comment by hoffmann-stephen** [comment]
@JohnnyUrosevic  Could you make a pull request for this?  Would really like to play around with this but can't even get started

**Comment by mastercoms** [comment]
Fixed in baf9222607.
